config/environments/development.rb

```diff
@@ -35,6 +35,19 @@
   # Raises helpful error messages.
   config.assets.raise_runtime_errors = true
 
+  if ENV['SENDGRID_USERNAME'] && ENV['SENDGRID_PASSWORD']
+    # SMTP / Sendgrid settings
+    ActionMailer::Base.smtp_settings = {
+      address:        'smtp.sendgrid.net',
+      port:           '587',
+      authentication: :plain,
+      user_name:      ENV['SENDGRID_USERNAME'],
+      password:       ENV['SENDGRID_PASSWORD'],
+      domain:         'heroku.com',
+      enable_starttls_auto: true,
+    }
+  end
+
   # Raises error for missing translations
```

Comment:
@chrisdolendo did you add this just to test real emails rather than use the server log (which understandably can be kind of hard to read)? I should also show you the `EmailerPreview` class which makes it a lot easier to test these emails without having to actually send them off